config/checkstyle_input_suppressions.xml

```diff
@@ -864,14 +850,6 @@
             files="imports[\\/]importorder[\\/]InputImportOrderStaticGroupsTopSeparated.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="imports[\\/]importorder[\\/]InputImportOrderStaticGroupsUnordered.java"/>
```

Comment:
please remove all `ConfigCommentOnTopInputs ` and `StyleValidationCommentInInputs ` suppressions for `importorder`